build.gradle

```diff
@@ -2,23 +2,27 @@
 
 buildscript {
     repositories {
+        google()
         jcenter()
         maven {
```

Comment:
Remove this. It's a duplicate

Reply:
I think [this line](#https://github.com/onaio/kujaku/blob/8ca8b8d0602c821439144858326af3ff335be7d2/build.gradle#L7) is a duplicate of `google()`

Remove it, everything should work ok